main/RFXNames.h

```diff
@@ -173,8 +173,9 @@ enum _eHardwareTypes {
 	HTYPE_ZIBLUETCP,			//90
 	HTYPE_Yeelight,				//91
 	HTYPE_MySensorsMQTT,		//92
-	HTYPE_RaspberryPCF8574,				//93
-	HTYPE_XiaomiGateway,		//94
+	HTYPE_RaspberryPCF8574,		//93
+	HTYPE_PythonPlugin,			//94
+	HTYPE_XiaomiGateway,		//95
 	HTYPE_END
```

Comment:
I think you flipped the two hardware types
The python should be at the bottom correct?